charts/security-apps/values.yaml

```
@@ -153,7 +153,7 @@ secretsStoreCsiDriver:
   # secretsStoreCsiDriver.chart -- Chart
   chart: "secrets-store-csi-driver"
     # secretsStoreCsiDriver.targetRevision -- [vault-csi-provider Helm chart](https://github.com/kubernetes-sigs/secrets-store-csi-driver/blob/master/charts/secrets-store-csi-driver) version
-  targetRevision: "0.0.*"
+  targetRevision: "0.1.*"
   # secretsStoreCsiDriver.values -- Helm values
   # @default -- [upstream values](https://github.com/kubernetes-sigs/secrets-store-csi-driver/blob/master/charts/secrets-store-csi-driver/values.yaml)
   values: {}
```

Maybe relevant, [`syncSecret.enabled`](https://github.com/kubernetes-sigs/secrets-store-csi-driver/blob/master/charts/secrets-store-csi-driver/values.yaml#L192-L194) is being deprecated (by deafult set to false) in `v0.1.X`, and I think we would need it, it is described in [here](https://github.com/kubernetes-sigs/secrets-store-csi-driver/releases/tag/v0.1.0), probably good to add it? I add a suggestion just in case:

```suggestion
  values:
    # Install RBAC roles and bindings required for K8S Secrets syncing if true
    syncSecret:
      enabled: true
```

---

infra-apps never specify values but i should add it to the corresponding example

---

Added an example in `charts/security-apps/examples/secrets-store-csi-driver.yaml`.